client/src/js/services/GeneralLedgerService.js

```diff
@@ -0,0 +1,26 @@
+angular.module('bhima.services')
+  .service('GeneralLedgerService', GeneralLedgerService);
+
+// Dependencies injection
+GeneralLedgerService.$inject = ['$http', 'util'];
+
+/**
+ * General Ledger Service
+ * This service is responsible of all process with the General ledger
+ */
+function GeneralLedgerService($http, util) {
```

Comment:
This is a perfect use case for the `PrototypeApiService` (#pro) which is well tested.  Instead of 15 lines of code, this could be:
```js
function GeneralLedgerService(Api) {
  return new Api('/general_ledger');
}
```